compiler/docgen.nim

```diff
@@ -28,24 +28,50 @@ const
   DocColOffset = "## ".len  # assuming that a space was added after ##
 
 type
+  ItemFragment = object  ## A fragment from each item will be eventually
+                         ## constructed by converting `rst` fields to strings.
+    case isRst: bool
+    of true:
+      rst: PRstNode
+    of false:
```

Comment:
add a comment indicating what it corresponds to if isRst is false

Reply:
added.